README.md

```diff
@@ -42,6 +42,8 @@ In order to stop the containers, run
 docker-compose down
 ````
 
+To populate the bank holidays table locally , call ``http://localhost:8082/bankHoliday/refresh``.
```

Comment:
Does this have a cron job associated with it somewhere? Would need to be a K8 one for now due to our distributed system.

Reply:
I've added this now